### DISCUSSION TITLE
Mixed models: conditional or marginal R2?

### DISCUSSION THREAD

**Discussion by MarcRieraDominguez** [jluchman/domir#10] [General]
Good afternoon!
I am running dominance analysis on a mixed-effects beta regression, and on binomial GLMM, and I would like your opinion on which would be a suitable metric of model quality.

My research question focuses on the fixed factors, while the random effects are a necessity of the non-independence of the data. Therefore, I would choose the marginal R2 over the conditional R2. Is the choice of marginal over conditional R2 conceptually sound?

However, the differences between using the marginal and conditional R2 can be substantial. In my mixed-effects beta regression, the R2 marginal is four times less than the R2 conditional; and the standardized general dominance of a predictor changed from 0.4 to 0.24 (marginal vs conditional), thus changing its ranking. Is the impact of the choice of R2 more impactful with increasing "importance" of random efects?

I read in Azen & Budescu (2003, DOI: 10.1037/1082-989X.8.2.129) that "any measure of model fit that is a monotone (increasing or decreasing) function of the model’s error sum of squares (SSE) would yield the same dominance pattern.", citing a formal proof for adjusted R2, Akaike's Information Criterion and Cp. Maybe the SSE is not meaningful for mixed-models involving proportions, hence the stark change in results for a mixed-effects beta regression.

In case it matters, I fit the mixed models with `glmmTMB::glmmTMB`, calculate marginal and conditional R2 with `MuMIn::r.squaredGLMM()`, and implement dominance analysis with `domir::domir()`.

Thanks!

**Comment by fmg-jluchman** [comment]
Hi Marc,

  Would recommend the marginal version of these $R^2$-s for any multilevel model as the conditional $R^2$-s always include the random effects _as_ an explanatory factor, along with the variability produced by the coefficient estimates, in the numerator of the computation where the marginal version just includes variation produced by the coefficient estimates.

  Suppose there could be situations where the conditional would be of use for inferring importance. More than likely though, most research questions are best applied to the marginal version for dominance analysis.

- joe

**Reply by MarcRieraDominguez** [reply]
Great! Thanks once more for your insights!

**Comment by MarcRieraDominguez** [comment]
Going back to this after a while, I ended up implementing dominance analysis with marginal R2. See the paper here: https://doi.org/10.1111/1365-2745.70092; and code here: https://doi.org/10.34810/data1716
Thanks a million Joe for your help!! :)
